### PR TITLE
Backports for 0.3.1-rc3

### PR DESCRIPTION
**Pull request by ax3l**
Backports for `0.3.1-rc3`:

- `tbg: --help` exit 0 #2213
- Density: Param Gaussian Density #2214
- cmakeFlags: Escape Lists #2183
- Tools: All C++11 #2194
- Boost Transient Deps: date_time, chrono, atomic #2195
- CMake: Use Imported Targets Zlib, Boost #2193
- bump version number to `0.3.1-rc3`

Note:
- [x] compile suite has changed, manual compile test on hypnos was successful

### CODE REVIEW

File: src/picongpu/CMakeLists.txt [hunk 1]
```diff
@@ -142,18 +142,35 @@ set(LIBS ${LIBS} ${mallocMC_LIBRARIES})
 
 find_package(ZLIB REQUIRED)
 include_directories(SYSTEM ${ZLIB_INCLUDE_DIRS})
-set(LIBS ${LIBS} ${ZLIB_LIBRARIES})
+set(LIBS ${LIBS} ZLIB::ZLIB)
```
Comment (ax3l) [inline src/picongpu/CMakeLists.txt]: note: [this target](https://github.com/Kitware/CMake/blob/v3.3.0/Modules/FindZLIB.cmake) does already exist in CMake 3.3+

**Comment by ax3l** [comment]
@PrometheusPi manual compile test succeeded :)

**Comment by PrometheusPi** [comment]
Ohh my god - I will merge now with the compile suite telling me not to do so 😱

File: src/picongpu/CMakeLists.txt [hunk 2]
```diff
@@ -142,18 +142,35 @@ set(LIBS ${LIBS} ${mallocMC_LIBRARIES})
 
 find_package(ZLIB REQUIRED)
 include_directories(SYSTEM ${ZLIB_INCLUDE_DIRS})
```
Comment (ax3l) [inline src/picongpu/CMakeLists.txt]: note: forgot to remove this line in the backport

**Pull request merged**